config/locales/en.yml

```diff
@@ -110,7 +110,7 @@ en:
         We did not find an account with the username or email you provided.
       found_account_but_it_has_no_emails: >-
         We found your account but can't send you an email because your account
-        doesn't have any email addresses listed. Please contact support for
+        doesn't have any email addresses listed. Please contact Support for
```

Comment:
marcom has asked us to say "Support" instead of "support" or "customer service" or "customer support".